tools/RedisConfigs/start-single.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+echo Starting Single: 
+pushd %~dp0\Basic
+echo   Master: 6380
+@start "Redis (Master): 6380" /min ..\3.0.503\redis-server.exe master-6380.conf
```

Comment:
@catcherwong without this i could not debug RedisCachingProviderTest locally. Tests uses port 6380, but in start basic this port is assigned to slave service.
Maybe it would be better to change port in tests, but I'm not sure how it works in AppVeyor.

Reply:
AppVeyor using docker to setup the redis.